core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-background-columns/template-builder-background-columns.component.scss

```diff
@@ -1,13 +1,17 @@
 @use "variables" as *;
 
-$columns-padding: 3.2rem;
-
 :host {
     grid-template-columns: repeat(12, minmax(0, 1fr));
     display: grid;
 }
 
 .column {
     height: 100%;
+    border-radius: $border-radius-sm;
+    background-color: $white;
+    transition: background-color 0.35s ease-in-out;
+}
+
+.visible {
```

Comment:
This component has no need to handle it shows... it will show all the time is the parent who needs to show and hide.